src/Aura/Languages.hs

```diff
@@ -339,9 +340,10 @@ buildFail_6 = \case
 -- NEEDS TRANSLATION
 buildFail_7 :: String -> Language -> String
 buildFail_7 (bt -> p) = \case
-    Japanese -> p ++ "の作成スクリプトを収得できなかった。"
-    Indonesia-> "Gagal mendapatkan skrip untuk " ++ p ++ "."
-    _        -> "Failed to obtain build scripts for " ++ p ++ "."
+    Japanese  -> p ++ "の作成スクリプトを収得できなかった。"
+    German    -> "Herunterladen der Build-Skripte für " ++ p ++ " fehlgeschlagen."
```

Comment:
'herunterladen' is German for 'download'. Other programs (e.g. apt-get) use 'holen' instead which is a more direct translation. However, it sounds a lot better to use 'herunterladen' in contexts where 'obtain' is used interchangably with 'download'. Also, 'Build-Skripte' is kind of a semi-translation, but way better than 'Bau-Skripte' (using the verb 'bauen' as translation for 'build' here like in translations before). The latter would sound ridiculous and wouldn't be better understood IMHO – the translation 'bauen' is already a poor one that only exists because Germanizing 'build' ('builden') sounds even worse.